src/wp-admin/includes/update-core.php

```diff
@@ -18,6 +18,8 @@
  */
 global $_old_files;
 
+require_once ABSPATH . 'wp-includes/Requests/Exception.php';
```

Comment:
```suggestion
/**
 * Preload the "from" version file(s) into memory before the upgrade
 * process replaces the files.
 * 
 * Why? Requests 2.0.0 (introduced in WordPress 5.9.0) introduced
 * namespaces. If a class is needed after its file is upgraded but
 * the class is not already available in memory, a fatal error is thrown.
 * Why? The fully-qualified name of the needed class is no longer
 * available in the filesystem. 
 * 
 * @since 5.9.0
 */
require_once ABSPATH . 'wp-includes/Requests/Exception.php';
```

Reply:
@costdev @pbiron what do you think of this explanation 👆? Attempting to make it clear about why files need to be preloaded vs. making it singularly specific to the one file for this ticket. Why? To give flexibility should other files also need to preloaded.

Reply:
how about this slightly different wording:

```
/**
 * Preload files from the current version into memory that may be needed
 * later during the update process, and may not be available after files
 * from the updated version have been installed.
 *
 * Why? Requests 2.0.0 (introduced in WordPress 5.9.0) introduced
 * namespaces. If a class is needed after its file is upgraded but
 * the class is not already available in memory, a fatal error is thrown
 * because the fully-qualified name of the needed class is no longer
 * available in the filesystem.
 *
 * @since 5.9.0
 */
```

I think it's a little easier to read.

The other suggestion I'd have is to put this comment and the `require_once` above the comment that comes before `global $_old_files;`

Reply:
Even better. Thanks @pbiron 

Done in https://github.com/WordPress/wordpress-develop/pull/1999/commits/a6279e6cfaeba04e6143747cb5100078e120a6f8 and https://github.com/WordPress/wordpress-develop/pull/1999/commits/857491c729d16f1c1cd1609859f799985c337561